### PR TITLE
Update app.js

### PR DESCRIPTION
**Pull request by chrjean**
Fix VLC path when Install dir is not specified in 'HKLM/Software/Wow6432Node/VideoLAN/VLC'

### CODE REVIEW

File: app.js [hunk 1]
```diff
@@ -204,6 +204,9 @@ var ontorrent = function (torrent) {
       if (process.arch === 'x64') {
         try {
           key = registry('HKLM/Software/Wow6432Node/VideoLAN/VLC')
+          if (!key['InstallDir']){
+            throw "no install dir";
```
Comment (mafintosh) [inline app.js]: could made this into `throw new Error('no install dir')` instead?

**Comment by chrjean** [comment]
yep, done :)

**Comment by mafintosh** [comment]
woooooot

**Comment by mafintosh** [comment]
0.32.4

**Pull request merged**